### ISSUE TITLE
Add rule for requiring variable type

### ISSUE THREAD

**Issue by jadnco** [gajus/eslint-plugin-flowtype#19]
As title states...

**Comment by gajus** [comment]
Please elaborate.

**Comment by jadnco** [comment]
@gajus This would be similar to `require-parameter-type` but for variables. All variables (`let`, `var`, `const`) should require a type.

ie.

```js
/* @flow */

const test: string = 'Hello World'; // good
const testTwo = 'Hello Mars'; // bad
```

**Comment by gajus** [comment]
It is not something that I would use, but PR is welcome.
